src/JSInterop/Microsoft.JSInterop/src/JSRuntimeExtensions.cs

```diff
@@ -28,76 +28,76 @@ public static async ValueTask InvokeVoidAsync(this IJSRuntime jsRuntime, string
 
     /// <summary>
     /// Invokes the specified JavaScript function asynchronously.
-    /// <para>
-    /// <see cref="JSRuntime"/> will apply timeouts to this operation based on the value configured in <see cref="JSRuntime.DefaultAsyncTimeout"/>. To dispatch a call with a different timeout, or no timeout,
-    /// consider using <see cref="IJSRuntime.InvokeAsync{TValue}(string, CancellationToken, object[])" />.
-    /// </para>
     /// </summary>
     /// <param name="jsRuntime">The <see cref="IJSRuntime"/>.</param>
-    /// <typeparam name="TValue">The JSON-serializable return type.</typeparam>
     /// <param name="identifier">An identifier for the function to invoke. For example, the value <c>"someScope.someFunction"</c> will invoke the function <c>window.someScope.someFunction</c>.</param>
+    /// <param name="cancellationToken">
+    /// A cancellation token to signal the cancellation of the operation. Specifying this parameter will override any default cancellations such as due to timeouts
+    /// (<see cref="JSRuntime.DefaultAsyncTimeout"/>) from being applied.
+    /// </param>
     /// <param name="args">JSON-serializable arguments.</param>
-    /// <returns>An instance of <typeparamref name="TValue"/> obtained by JSON-deserializing the return value.</returns>
-    public static ValueTask<TValue> InvokeAsync<[DynamicallyAccessedMembers(JsonSerialized)] TValue>(this IJSRuntime jsRuntime, string identifier, params object?[]? args)
+    /// <returns>A <see cref="ValueTask"/> that represents the asynchronous invocation operation.</returns>
+    public static async ValueTask InvokeVoidAsync(this IJSRuntime jsRuntime, string identifier, CancellationToken cancellationToken, params object?[]? args)
     {
         ArgumentNullException.ThrowIfNull(jsRuntime);
 
-        return jsRuntime.InvokeAsync<TValue>(identifier, args);
+        await jsRuntime.InvokeAsync<IJSVoidResult>(identifier, cancellationToken, args);
```

Comment:
Don't remember exactly, but I think this will cause an extra allocation (at least for the state machine)

Reply:
The diffs in this file are because I had previously added some new APIs, but then later removed them when prototyping option 2, but the order of some methods was different as a result. So this is diffing two lines from two different methods.